test/suites/config.sh

```diff
@@ -1,3 +1,81 @@
+ensure_removed() {
+  bad=0
+  lxc exec foo -- stat /dev/lxdkvm && bad=1 || true
+  if [ "${bad}" -eq 1 ]; then
+    echo "device should have been removed; $*"
+    false
+  fi
+}
+
+dounixdevtest() {
+    lxc start foo
+    lxc config device add foo kvm unix-char $*
+    lxc exec foo -- stat /dev/lxdkvm
+    lxc exec foo reboot
+    lxc exec foo -- stat /dev/lxdkvm
+    lxc restart foo --force
+    lxc exec foo -- stat /dev/lxdkvm
+    lxc config device remove foo kvm
+    ensure_removed "was not hot-removed"
+    lxc exec foo reboot
+    ensure_removed "removed device re-appeared after container reboot"
+    lxc restart foo --force
+    ensure_removed "removed device re-appaared after lxc reboot"
+    lxc stop foo --force
+}
+
+testunixdevs() {
+  rm -rf /dev/lxdkvm || true
+  if mknod /dev/lxdkvm c 10 232; then
+    echo "Testing /dev/lxdkvm"
+    dounixdevtest path=/dev/lxdkvm
+    rm -f /dev/lxdkvm
+  fi
+  echo "Testing /dev/lxdkvm 10 232"
+  dounixdevtest path=/dev/lxdkvm major=10 minor=232
+}
+
+ensure_fs_unmounted() {
+  bad=0
+  lxc exec foo -- stat /mnt/hello && bad=1 || true
+  if [ "${bad}" -eq 1 ]; then
+    echo "device should have been removed; $*"
+    false
+  fi
+}
+
+testloopmounts() {
+  lpath=`losetup -f` || { echo "no loop support"; return; }
+  echo "$lpath" >> ${TEST_DIR}/loops
+  loop=`basename $lpath`
+  loopfile=`mktemp -p ${TEST_DIR} loop_XXX`
+  dd if=/dev/zero of=$loopfile bs=1M seek=200 count=1
+  mkfs.ext4 $loopfile
```

Comment:
Weird, does wily add that one automatically?  I had almost typed -f originally but noticed i didn't need it so ...